src/components/base/JAssetSymbol/jAssetSymbol.scss

```diff
@@ -15,16 +15,23 @@
   font-size: inherit;
 
   &.-symbol-not-listed::before {
-    @include font;
+    @include font(14px, 38px);
 
-    display: block;
-    content: attr(data-symbol);
     position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+    box-sizing: border-box;
+    content: attr(data-symbol);
+    overflow: hidden;
+    width: 40px;
+    height: 40px;
+    border: 1px solid $main-black-10;
+    border-radius: 50%;
+    font-style: normal;
     font-weight: 600;
-    font-size: inherit;
+    text-align: center;
+    text-transform: uppercase;
+    text-overflow: ellipsis;
```

Comment:
Didn't we decide earlier there is no ellipsis? Maybe better to hide overflow symbols?

Reply:
It just a defense by dumb situations with too long texts. But, I guess, your proposal could better